packages/dev/docs/src/types.js

```diff
@@ -222,9 +222,9 @@ export function InterfaceType({properties, showRequired, showDefault}) {
       <thead>
         <tr>
           <td className={tableStyles['spectrum-Table-headCell']}>Name</td>
-          <td className={tableStyles['spectrum-Table-headCell']}>Type</td>
+          <td className={tableStyles['spectrum-Table-headCell']} style={{'width': '30%'}}>Type</td>
```

Comment:
Because Menu props children type definition try to take over the entire table I added this to make things look better.